pkg/minikube/cluster/cluster.go

```diff
@@ -379,6 +380,40 @@ func GetHostLogs(api libmachine.API) (string, error) {
 	return s, nil
 }
 
+func getHyperVSwitch() (string, error) {
+	driverConfig, err := ioutil.ReadFile(filepath.Join(constants.GetMinipath(), "machines", "minikube", "config.json"))
+	if err != nil {
+		return "", errors.Wrap(err, "Error getting hyperv virtual switch config")
+	}
+	re := regexp.MustCompile("\"VSwitch\": \"(.*?)\",")
+	match := re.FindStringSubmatch(string(driverConfig))
+	return match[1], nil
+}
+
+// Mount9pHost runs the mount command from the 9p client on the VM to the 9p server on the host
+func Mount9pHost(api libmachine.API) error {
+	host, err := CheckIfApiExistsAndLoad(api)
+	if err != nil {
+		return errors.Wrap(err, "Error checking that api exists and loading it")
+	}
+	var hypervSwitch string
+	if host.DriverName == "hyperv" {
+		hypervSwitch, err = getHyperVSwitch()
```

Comment:
Could you unmarshall the driver into the real hypervdriver struct and get the value that way, instead of using regex?